test/unit_tests/events/aws_v3_test.js

```diff
@@ -0,0 +1,37 @@
+const { SNSClient, PublishCommand } = require('@aws-sdk/client-sns');
+const { expect } = require('chai');
+const epsagon = require('../../../src/index');
+const tracerObj = require('../../../src/trace_object.js');
+
+
+describe('aws sdk v3 sns-client tests', () => {
+    beforeEach(() => {
+        epsagon.init({
+            token: process.env.EPSAGON_TOKEN,
+            appName: 'aws-sdk-v3-test',
+            metadataOnly: false,
+        });
+    });
+
+
+    it('test instrumentation of publish to some non-existing sns', async () => {
+        function publishSns() {
+            const snsClient = new SNSClient({ region: 'ap-southeast-1' });
```

Comment:
I suggest having the 'ap-southeast-1' as a const, and having a test_consts file -> in order to make tests more generic and readable